src/app/components/applications/SSV/MyAccount/components/SingleOperator/SingleOperator.tsx

```diff
@@ -88,7 +82,7 @@ const SingleOperator = () => {
 
   // @ts-ignore
   const { logo, validators_count, fee, performance } = operator || {};
-  const validator30dPerformance = operator ? performance['30d'] : 0;
+  const validator30dPerformance = operator ? performance['30d'] : 0; // TODO Why if in useEffect in line 50 operator is null checked?
```

Comment:
@Chris-ssvlabs  if you mean why do we also null check here, it's because this code will run before the useEffect and it can break

Reply:
`validator30dPerformance` is used only for display it's value in line 170.
But only case when `operator` is there it will be displayed.
If other case, when not `operator` User will be redirected.